### DISCUSSION TITLE
25+ Stable Diffusion Tutorials - Guides - DreamBooth - Textual Inversion - LoRA - ControlNet - Style Transfer - RunPod - Animation

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [TheLastBen/fast-stable-diffusion#1582] [Show and tell]
## Hello dear TheLastBen. You are an amazing contributor to the Stable Diffusion community. 

# Expert-Level Tutorials on Stable Diffusion: Master Advanced Techniques and Strategies

Greetings everyone. I am Dr. Furkan Gözükara. I am an Assistant Professor in Software Engineering department  of a private university (have PhD in Computer Engineering). My professional programming skill is unfortunately C# not Python :) 

My linkedin : https://www.linkedin.com/in/furkangozukara/

### Our discord to get more help : https://discord.com/servers/software-engineering-courses-secourses-772774097734074388

I am keeping this list up-to-date. I got upcoming new awesome video ideas. Trying to find time to do that.

### All videos have manually fixed subtitles and properly prepared video chapters. You can watch with these perfect subtitles or look for the chapters you are interested in.

Since my profession is teaching, I usually do not skip any of the important parts. Therefore, you may find my videos a little bit longer.

Playlist link on YouTube: [Stable Diffusion Tutorials, Automatic1111 and Google Colab Guides, DreamBooth, Textual Inversion / Embedding, LoRA, AI Upscaling, Pix2Pix, Img2Img](https://www.youtube.com/watch?v=mnCY8uM7E50&list=PL_pbwdIyffsmclLl0O144nQRnezKlNdx3)

1.) Automatic1111 Web UI - PC - Free
[Easiest Way to Install & Run Stable Diffusion Web UI on PC by Using Open Source Automatic Installer](https://www.youtube.com/watch?v=AZg6vzWHOTA)
[![image](https://user-images.githubusercontent.com/19240467/218344261-aa236e18-152f-4287-b4fd-fa09c8f57a3f.png)](https://www.youtube.com/watch?v=AZg6vzWHOTA)

2.) Automatic1111 Web UI - PC - Free
[How to use Stable Diffusion V2.1 and Different Models in the Web UI - SD 1.5 vs 2.1 vs Anything V3](https://www.youtube.com/watch?v=aAyvsX-EpG4)
[![image](https://user-images.githubusercontent.com/19240467/218344276-af8f2fa2-4fdb-4454-9c8f-bd3ef4d2c92a.png)](https://www.youtube.com/watch?v=aAyvsX-EpG4)

3.) Automatic1111 Web UI - PC - Free
[Zero To Hero Stable Diffusion DreamBooth Tutorial By Using Automatic1111 Web UI - Ultra Detailed](https://www.youtube.com/watch?v=Bdl-jWR3Ukc)
[![image](https://user-images.githubusercontent.com/19240467/218344301-04f91cf4-fa35-4975-8c3d-9951c765839a.png)](https://www.youtube.com/watch?v=Bdl-jWR3Ukc)

4.) Automatic1111 Web UI - PC - Free
[DreamBooth Got Buffed - 22 January Update - Much Better Success Train Stable Diffusion Models Web UI](https://www.youtube.com/watch?v=KwxNcGhHuLY)
[![image](https://user-images.githubusercontent.com/19240467/218344369-97b68dd7-732d-4ca3-9acc-a87984ebe0f0.png)](https://www.youtube.com/watch?v=KwxNcGhHuLY)

5.) Automatic1111 Web UI - PC - Free
[How to Inject Your Trained Subject e.g. Your Face Into Any Custom Stable Diffusion Model By Web UI](https://www.youtube.com/watch?v=s25hcW4zq4M)
[![image](https://user-images.githubusercontent.com/19240467/218344509-01d70965-aeea-4096-bc29-7a005b4d47a6.png)](https://www.youtube.com/watch?v=s25hcW4zq4M)
 
6.) Automatic1111 Web UI - PC - Free
[How To Do Stable Diffusion LORA Training By Using Web UI On Different Models - Tested SD 1.5, SD 2.1](https://www.youtube.com/watch?v=mfaqqL5yOO4)
[![image](https://user-images.githubusercontent.com/19240467/218344459-bd4554b0-b57b-4079-aaea-ed93d8be95ed.png)](https://www.youtube.com/watch?v=mfaqqL5yOO4)

7.) Automatic1111 Web UI - PC - Free
[8 GB LoRA Training - Fix CUDA & xformers For DreamBooth and Textual Inversion in Automatic1111 SD UI](https://www.youtube.com/watch?v=O01BrQwOd-Q)
[![image](https://user-images.githubusercontent.com/19240467/218344491-52ac51d8-6556-4abc-b2fb-d640a46c48a2.png)](https://www.youtube.com/watch?v=O01BrQwOd-Q)

8.) Automatic1111 Web UI - PC - Free
[How To Do Stable Diffusion Textual Inversion (TI) / Text Embeddings By Automatic1111 Web UI Tutorial](https://www.youtube.com/watch?v=dNOpWt-epdQ)
[![image](https://user-images.githubusercontent.com/19240467/218344538-d5f0329d-b0e9-44ed-aaf0-5e4bb134afb7.png)](https://www.youtube.com/watch?v=dNOpWt-epdQ)

9.) Automatic1111 Web UI - PC - Free
[How To Generate Stunning Epic Text By Stable Diffusion AI - No Photoshop - For Free - Depth-To-Image](https://www.youtube.com/watch?v=TBq1bhY8BOc)
[![image](https://user-images.githubusercontent.com/19240467/218344579-fda1e9b8-a810-48af-9dcb-f47e87afee9e.png)](https://www.youtube.com/watch?v=TBq1bhY8BOc)

10.) Python Code - Hugging Face Diffusers Script - PC - Free
[How to Run and Convert Stable Diffusion Diffusers (.bin Weights) & Dreambooth Models to CKPT File](https://www.youtube.com/watch?v=-6CA18MS0pY)
[![image](https://user-images.githubusercontent.com/19240467/218344677-3f812cf3-db37-4ccb-8f81-99b8a1d5ef00.png)](https://www.youtube.com/watch?v=-6CA18MS0pY)

11.) NMKD Stable Diffusion GUI - Open Source - PC - Free
[Forget Photoshop - How To Transform Images With Text Prompts using InstructPix2Pix Model in NMKD GUI](https://www.youtube.com/watch?v=EPRa8EZl9Os)
[![image](https://user-images.githubusercontent.com/19240467/218344868-3232f875-b2c5-4caa-b59b-9d0fd683c06b.png)](https://www.youtube.com/watch?v=EPRa8EZl9Os)

12.) Google Colab Free - Cloud - No PC Is Required
[Transform Your Selfie into a Stunning AI Avatar with Stable Diffusion - Better than Lensa for Free](https://www.youtube.com/watch?v=mnCY8uM7E50)
[![image](https://user-images.githubusercontent.com/19240467/218344900-286cded5-0171-4b9e-9354-7adf4bada612.png)](https://www.youtube.com/watch?v=mnCY8uM7E50)

13.) Google Colab Free - Cloud - No PC Is Required
[Stable Diffusion Google Colab, Continue, Directory, Transfer, Clone, Custom Models, CKPT SafeTensors](https://www.youtube.com/watch?v=kIyqAdd_i10)
[![image](https://user-images.githubusercontent.com/19240467/218344930-95956805-6a6e-46ee-8885-64043246d79b.png)](https://www.youtube.com/watch?v=kIyqAdd_i10)

14.) Automatic1111 Web UI - PC - Free
[Become A Stable Diffusion Prompt Master By Using DAAM - Attention Heatmap For Each Used Token - Word](https://www.youtube.com/watch?v=XiKyEKJrTLQ)
[![image](https://user-images.githubusercontent.com/19240467/218345146-54076e5d-230a-4774-8d6a-8358cbd15f78.png)](https://www.youtube.com/watch?v=XiKyEKJrTLQ)

15.) Python Script - Gradio Based - ControlNet - PC - Free
[Transform Your Sketches into Masterpieces with Stable Diffusion ControlNet AI - How To Use Tutorial](https://www.youtube.com/watch?v=YJebdQ30UZQ)
[![image](https://user-images.githubusercontent.com/19240467/218345328-ada437bf-5eb4-478e-a951-84486a42995d.png)](https://www.youtube.com/watch?v=YJebdQ30UZQ)

16.) Automatic1111 Web UI - PC - Free
[Sketches into Epic Art with 1 Click: A Guide to Stable Diffusion ControlNet in Automatic1111 Web UI](https://www.youtube.com/watch?v=vhqqmkTBMlU) 
[![image](https://user-images.githubusercontent.com/19240467/218806127-c84d1ff8-d5bb-41b0-bdef-6922568792b9.png)
](https://www.youtube.com/watch?v=vhqqmkTBMlU)

17.) RunPod - Automatic1111 Web UI - Cloud - Paid - No PC Is Required
[Ultimate RunPod Tutorial For Stable Diffusion - Automatic1111 - Data Transfers, Extensions, CivitAI](https://www.youtube.com/watch?v=QN1vdGhjcRc) 
[![image](https://user-images.githubusercontent.com/19240467/219958249-82ecb925-901b-4f87-b776-f592b0f5eaad.png)](https://www.youtube.com/watch?v=QN1vdGhjcRc)

18.) Automatic1111 Web UI - PC - Free
[Fantastic New ControlNet OpenPose Editor Extension & Image Mixing - Stable Diffusion Web UI Tutorial](https://youtu.be/iFRdrRyAQdQ) 
[![image](https://user-images.githubusercontent.com/19240467/220776337-3abce5a3-bb17-4240-8400-4e633562ecc8.png)](https://youtu.be/iFRdrRyAQdQ)

19.) Automatic1111 Web UI - PC - Free
[Automatic1111 Stable Diffusion DreamBooth Guide: Optimal Classification Images Count Comparison Test](https://youtu.be/Tb4IYIYm4os) 
[![image](https://user-images.githubusercontent.com/19240467/221384116-e42d6f37-a068-4a2a-9bda-11ac47f33faa.png)](https://youtu.be/Tb4IYIYm4os)

20.) Automatic1111 Web UI - PC - Free
[Epic Web UI DreamBooth Update - New Best Settings - 10 Stable Diffusion Training Compared on RunPods](https://youtu.be/sRdtVanSRl4) 
[![image](https://user-images.githubusercontent.com/19240467/222991604-ceed12bc-0bc9-4f16-82fe-e6779132e00c.png)](https://youtu.be/sRdtVanSRl4)

21.) Automatic1111 Web UI - PC - Free
[New Style Transfer Extension, ControlNet of Automatic1111 Stable Diffusion T2I-Adapter Color Control](https://youtu.be/tXaQAkOgezQ)
[![image](https://user-images.githubusercontent.com/19240467/223283277-eaaf6e53-df43-40ac-8096-c08f9a14cc8d.png)
](https://youtu.be/tXaQAkOgezQ)

22.) Automatic1111 Web UI - RunPod - Paid
[How To Install New DreamBooth Extension On RunPod - Automatic1111 Web UI - Stable Diffusion](https://youtu.be/oI0GEnrySS0)
[![image](https://user-images.githubusercontent.com/19240467/224442527-37e02628-dfc9-45ad-8e28-8735bbf237b4.png)
](https://youtu.be/oI0GEnrySS0)

23.) Automatic1111 Web UI - PC - Free
[Generate Text Arts & Fantastic Logos By Using ControlNet Stable Diffusion Web UI For Free Tutorial](https://youtu.be/C_mJI4U23nQ)
[![image](https://user-images.githubusercontent.com/19240467/224442765-ba241f71-b412-4f5b-bf39-506e9682e336.png)
](https://youtu.be/C_mJI4U23nQ)

24.) Automatic1111 Web UI - PC - Free
For downgrade to older version if you don't like Torch 2 : first delete venv, let it reinstall, then activate venv and run this command pip install -r "path_of_SD_Extension\requirements.txt"
[How To Install New DREAMBOOTH & Torch 2 On Automatic1111 Web UI PC For Epic Performance Gains Guide](https://youtu.be/pom3nQejaTs)
[![image](https://user-images.githubusercontent.com/19240467/226115542-72db7e7e-cee0-4e3a-82c4-12348e2b237e.png)
](https://youtu.be/pom3nQejaTs)

25.) Automatic1111 Web UI - PC - Free
[Training Midjourney Level Style And Yourself Into The SD 1.5 Model via DreamBooth Stable Diffusion](https://youtu.be/m-UVVY_syP0)
[![image](https://user-images.githubusercontent.com/19240467/226378438-fe70f09e-94a8-4d1d-9468-e44dca99aac7.png)
](https://youtu.be/m-UVVY_syP0)

**Comment by THX1139b** [comment]
Isn’t this the discussion section for TheLastBen’s Dreambooth and Automatic1111? I’m watching these Dreambooth tutorials and they are for something else, not this. TheLastBen’s Dreambooth is a separate Colab notebook with a different layout and settings altogether.

**Reply by FurkanGozukara** [reply]
> Isn’t this the discussion section for TheLastBen’s Dreambooth and Automatic1111? I’m watching these Dreambooth tutorials and they are for something else, not this. TheLastBen’s Dreambooth is a separate Colab notebook with a different layout and settings altogether.

I kind of agree but any tutorial made on Automatic1111 is useful for TheLastBen google colab. Am I incorrect?
